server/routes/orders.js

```diff
@@ -25,8 +25,6 @@ orderRoute.get('/:id', async(req, res, next) => {
             res.send(await Order.findByPk(req.params.id, {
                 include: [{
                     model: Item
```

Comment:
I pointed this out on the axios call, but this route sends back an order from a primary key. Your front-end axios call is sending a user id, not the order id.

If we're going to query for cart by user.id, I think we need a route that does this:

`res.send(await Order.findOne({
    where: {
        userId: req.params.id,
        status: 'cart',
    },
    include: ...`